### PR TITLE
Added I'll Make A Man Out Of You

### PR DESCRIPTION
**Pull request by SankalpSh**
- [x] 🔍 I have searched the `data.js` file and confirmed I am not adding a duplicate entry, unless it is a different version of a song.
- [x] 🖍️ I have placed the new song(s) in alphabetical order based on `title` inside of the `data.js` file. 
- [x] 🖍️ The `artist(s)` and `featured artist(s)` are all inside of square brackets `[ ]` and each are individually wrapped in quotation marks and have a comma between each one. (such as submitting this `"artist": ["Lady Gaga","Bradley Cooper"]` and not this `"genres":["Lady Gaga,Bradley Cooper"]`).
- [x] 3️⃣ I have at least three emojis listed under `emojiImgs`
- [x] 6️⃣ I have a maximum of six emojis listed under `emojiImgs`.
- [x] 🖍️ I have added a link to an official music video or performance under `musicVideo`. If there is no official video, I have added an empty string.
- [x] 👍 My pull request has a descriptive title (such as `Added Boy With Luv` or `Added all of Panic at the Disco songs`)
- [x] ⭐ My genres are all inside of square brackets `[ ]` and each are individually wrapped in quotation marks and have a comma between each one. (such as submitting this `"genres": ["k-pop","pop","dance"]` and not this `"genres":["k-pop, pop, dance"]`).
- [x] 📅 I have added a single year under `year`.

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -63,7 +63,17 @@ const emojiItems = [
         musicVideo: "https://www.youtube.com/watch?v=vc6vs-l5dkc",
         genres: ["emo", "pop", "punk"],
         year: 2005
-    },{
+    },
+    {
+        title: "I'll Make a Man Out of You",
+        artist: [" Donny Osmond"],
+        featuredArtist: [],
+        emojiImgs: "9️⃣🕒🌆",
```
Comment (brittanyrw) [inline data.js]: Please update the emojis for this song, it looks like you might have used a different song as a template and haven't updated the emojis yet.

File: data.js [hunk 2]
```diff
@@ -63,7 +63,17 @@ const emojiItems = [
         musicVideo: "https://www.youtube.com/watch?v=vc6vs-l5dkc",
         genres: ["emo", "pop", "punk"],
         year: 2005
-    },{
+    },
+    {
+        title: "I'll Make a Man Out of You",
+        artist: [" Donny Osmond"],
```
Comment (brittanyrw) [inline data.js]: Could you please remove the extra space in front of "Donny"?

File: data.js [hunk 3]
```diff
@@ -63,7 +63,17 @@ const emojiItems = [
...
+        featuredArtist: [],
+        emojiImgs: "9️⃣🕒🌆",
+        musicVideo: "https://www.youtube.com/watch?v=TVcLIfSC4OE",
+        genres: ["Children's"],
```
Comment (brittanyrw) [inline data.js]: Let's change the genre to just 'children' so we don't have to worry about the single quotation mark later when we start using these genres for filtering.

**Review by brittanyrw — CHANGES_REQUESTED** [review]
Hi @SankalpSh, it is looking good so far. 👍I have added a couple of comments for things that should be updated. When you get a chance, please make the updates and then we will review again!

File: data.js (hunk 1 above)
Comment (SankalpSh) [reply data.js]: Okay I will update the changes required and make a PR for the same

**Pull request merged**